apps/netting/config/config.py

```diff
@@ -0,0 +1,22 @@
+# Number of transactions
+NUMTX = 500000
+
+# Number of parties
+NUM_CLIENTS = 64
+
+# Average payment amount
+mu_tx = 10000.00
+mu_bal = 10000.00
+
+# Standard deviation
+sigma_tx = 100.00
+sigma_bal = 100.00
+
+# File name info
+tx_file_prefix = "party_"
+tx_file_suffix = "_tx_data.csv"
+
+bal_file_prefix = "party_"
+bal_file_suffix = "_bal_data.csv"
+
+NETTING_BASE_DIR = "/usr/src/HoneyBadgerMPC/apps/netting/"
```

Comment:
Is there a plan to have more files under `apps/netting/config/`? If not, `apps/netting/config.py` is sufficient and makes the code simpler.